src/runReporters.ts

```diff
@@ -4,20 +4,156 @@
 //
 // Not to be confused with https://www.npmjs.com/package/aws-lambda
 import { Handler } from 'aws-lambda'; // tslint:disable-line:no-implicit-dependencies
-import WebPageTest from 'webpagetest';
+import { collectReports } from './helpers/collectReports';
+import { format as formatDate } from 'date-fns';
 import bluebird from 'bluebird';
+import NodeGit from 'nodegit';
+import { join } from 'path';
+import Octokit from '@octokit/rest';
+import { renderReport } from './helpers/renderReport';
+import tmp from 'tmp';
+import { WebPageTestReporter } from './reporters/WebPageTestReporter';
+import { SecurityHeadersReporter } from './reporters/SecurityHeadersReporter';
+import { config } from './config';
+import { keyBy, mapValues } from 'lodash';
+import { writeFile } from './helpers/writeFile';
+import { stripIndents } from 'common-tags';
+import prettier from 'prettier';
 
-const API_KEY = process.env.WPT_API_KEY;
+// tslint:disable no-console
+// tslint:disable-next-line:max-func-body-length
+export const runReporters: Handler = async (_event, _context) => {
+  const urls = ['https://hollowverse.com', 'https://hollowverse.com/Tom_Hanks'];
+  const dateStr = formatDate(new Date(), 'YYYY-MM-DD');
 
-export const runReporters: Handler = async (_event, _context, callback) => {
-  const wpt = new WebPageTest('www.webpagetest.org', API_KEY);
+  const results = await bluebird.map(urls, async url => {
+    const reports = await collectReports({
+      url,
+      config,
+      reporters: [SecurityHeadersReporter, WebPageTestReporter],
+    });
 
-  const { data: { testId } } = await bluebird.fromNode<
-    WebPageTest.RunTestResponse
-  >(cb => wpt.runTest('https://hollowverse.com', cb));
+    return {
+      url,
+      raw: reports,
+      rendered: renderReport({
+        reports,
+      }),
+    };
+  });
 
-  // tslint:disable-next-line:no-console
-  console.log(await bluebird.fromNode(cb => wpt.getTestResults(testId, cb)));
+  const rawReports = mapValues(keyBy(results, r => r.url), r => r.raw);
+  let markdownReport = stripIndents`
+    Report for tests performed on ${dateStr}
+    ========================================
 
-  callback(null, 'Hello World');
+    ${results
+      .map(({ url, rendered }) => {
+        return stripIndents`
```

Comment:
I've always found this syntax weird ``foo`some template string` ``. I'll need to read up on it eventually 😄